core/blockchain_ext_test.go

```diff
@@ -898,33 +772,27 @@ func BuildOnVariousStages(t *testing.T, create createFunc) {
 	checkBlockChainState(t, blockchain, gspec, chainDB, create, checkState)
 }
 
-func EmptyBlocks(t *testing.T, create createFunc) {
+func EmptyBlocksTest(t *testing.T, create createFunc) {
```

Comment:
```suggestion
func EmptyBlocks(t *testing.T, create createFunc) {
```

Was this cherry-picked?